src/Utils/ModelUtil.vala

```diff
@@ -33,11 +33,42 @@
      * Return 0 on success.
      */
     public static int clone_from_model (
+        Lib.Items.Model source_model,
```

Comment:
Why do we need to split this? I envisioned clone_from_model to be generic between any two models. It's not a big deal either way, but It could be better?

Reply:
I couldn't find a better approach to handle the upcoming `paste in place` action.
Users can copy a node and then do a `paste` or `paste in place`.
The latter should position the cloned node in the same `pos_in_parent` of a selected item (just above), using its X & Y coordinates to position it in the canvas.
The former, should append the cloned node on top of everything and at the center of the current viewport.

I can try to keep it to one single method, without riddling with too many conditions as that seemed a bit messy.